src/templates/static/about/binary_in_numbers.jsx

```diff
@@ -45,7 +45,7 @@ const BinaryInNumbers = () => (
             <Numbers
                 className='employee'
                 header={it.L('Number of employees and contractors')}
-                text={it.L('We’ve grown in size over the years –– both in terms of manpower and offices. We currently have over 150 employees across four offices in Malaysia, Malta, and Japan. Further growth in headcount is expected this year to cater to the increased demand in our products and services.')}
+                text={it.L('We’ve grown in size over the years –– both in terms of manpower and offices. We currently have over 200 employees across three offices in Malaysia and Malta. Further growth in headcount is expected this year to cater to the increased demand in our products and services.')}
```

Comment:
I would suggest a placeholder for numbers (at least the `200`, not feasible for `three`) so translation doesn't break as we may regularly update this